### PR TITLE
Implement simple sphere obstacle

### LINKED ISSUE #8: Add oval object for particle collision

### PR DESCRIPTION
**Pull request by mitas1**
Closes #8

### CODE REVIEW

**Comment by mskrip** [comment]
> Go has no line length limit. Don't worry about overflowing a punched card. If a line feels too long, wrap it and indent with an extra tab.

From [Effective Go](https://golang.org/doc/effective_go.html)

File: objects.go [hunk 1]
```diff
@@ -32,6 +32,7 @@ type Particle struct {
 	sprite       pixel.Sprite // particle display image
 	lifespan     float64      // in s
 	alive        float64      // in s
+	collide      bool
```
[the remark this answers is not in the document]
Comment (mitas1) [reply objects.go]: Removed

File: physical-based-animations.go [hunk 2]
```diff
@@ -272,6 +308,7 @@ func run() {
 					sprite:       *particleSprite,
 					lifespan:     particleLife.value,
 					alive:        0.0,
+					collide:      false,
```
[the remark this answers is not in the document]
Comment (mitas1) [reply physical-based-animations.go]: Removed

**Comment by mitas1** [comment]
Particles resolves collision with Newton impact law is implemented and I know about line length limit in go but, I think wrapping lines up to 100 characters is really suitable for instance here in github if you have long line you need to horizontaly scroll.

**Comment by mskrip** [comment]
LGTM

**Pull request merged**